voight_kampff.gemspec

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {tests}/**/*`.split("\n")
   s.require_path  = 'lib'
 
-  s.add_dependency 'rack', ['>= 1.4', '< 3.0']
+  s.add_dependency 'rack', ['>= 1.4']
 
   s.add_development_dependency 'combustion', '~> 1.1'
-  s.add_development_dependency 'rails', '~> 5.2'
+  s.add_development_dependency 'rails', '>= 5.2'
```

Comment:
This is also dangerous: you have no major version lock, as I remember. Any major version, 7, 8 or 9, can be breaking for this gem. So please add (bump the maximum) only when tested and working.